### PR TITLE
Infrastructure improvements (gevent, postgis, datadog)

### PR DESCRIPTION
**Pull request by nickcatal**
- Add support for gevent workers, including properly patching postgres and adding proper pooling

- Installing postgis, which is needed for upcoming projects

- Updated datadog's tracer to the latest version, which offers a lot more visibility into app performance.

### CODE REVIEW

File: app/turnout/settings/core.py [hunk 1]
```diff
@@ -40,6 +41,9 @@
         "DATABASE_URL", default="pgsql://postgres:turnout@postgres:5432/turnout"
```
Comment (liewegas) [inline app/turnout/settings/core.py:41]: s/pgsql/postgis/

File: app/turnout/settings/core.py [hunk 2]
```diff
@@ -40,6 +41,9 @@
...
     )
 }
+DATABASES["default"]["ENGINE"] = "django_db_geventpool.backends.postgis"
```
Comment (liewegas) [inline app/turnout/settings/core.py:44]: ...can be used instead of this

File: app/turnout/settings/core.py [hunk 3]
```diff
@@ -78,7 +82,6 @@
     "rest_framework",
```
Comment (liewegas) [inline app/turnout/settings/core.py:82]: need to add to DJANGO_APPS:
https://github.com/vote/turnout/pull/142/commits/ef06f478eeef12cb5ea2f83ea9349b86559575fb#diff-ecc3487e6c1cffd39606c2a61272c195R70

File: app/turnout/settings/core.py (hunk 2 above)
Comment (nickcatal) [reply app/turnout/settings/core.py:44]: We need to use geventpool for our move to gevent. Can't use the built in postgis library

**Pull request merged**